guess.js

```diff
@@ -1,96 +0,0 @@
-const readline = require('readline');
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-    return new Promise((resolve, reject) => {
-        rl.question(questionText, resolve);
-    });
-}
-
-choice()
-async function choice() {
-    let start = await ask('human guess or computer guess? ')
-    if (start === 'human guess') {
-       return startComp()
-    }
-    else if (start === 'computer guess') {
-        return startHuman()
-    }
-}
-
-function randomInt(min, max) {
-    return Math.floor((Math.random()/2) * (max - min + 1) + min)
-}
-
-startComp();
-async function startComp() {
-    let game = await ask("Are you ready to start the game?")
-    let question = await ask('What number do you want your range till? ')
-    console.log('Alright I am thinking of a number ')
-    let min = 1
-    let max = question 
-    let compNumber = randomInt(min, max)
-    let playerGuess = await ask('What\'s your Guess? ')
-    playerGuess = parseInt(playerGuess)
-    if (playerGuess === compNumber) {
-        console.log('Damn you won!! Until next time foe!!')
-        process.exit()
-    }
-    else {
-        while (playerGuess !== compNumber) {
```

Comment:
having both of these lines here is a little redundant, you can just do your `while` loop and remove your `else` section. It will work the same way and cut down on the lines of code